scripts/workers/submission_worker.py

```diff
@@ -351,8 +351,16 @@ def run_submission(challenge_id, challenge_phase, submission, user_annotation_fi
                     challenge_phase_split = ChallengePhaseSplit.objects.get(challenge_phase=challenge_phase,
                                                                             dataset_split__codename=split_code_name)
                 except:
-                    stderr.write("ORGINIAL EXCEPTION: No such relation between Challenge Phase and DatasetSplit"
-                                 " specified by Challenge Host \n")
+                    try:
+                        dataset_split = DatasetSplit.objects.get(codename=split_code_name)
+                    except:
+                        error_message = "DatasetSplit with codename %s does not exist" % split_code_name
+                    else:
+                        error_message = "ORIGINAL EXCEPTION: No such relation between Challenge Phase (%s-%s) and" \
+                                        " DatasetSplit (%s-%s) specified by Challenge Host \n" % (
+                                            challenge_phase.id, challenge_phase.name, dataset_split.id,
+                                            dataset_split.name)
```

Comment:
This is too overwhelming!

Reply:
But this is what was asked in the issue: adding the Challenge Phase name and ID along with Dataset split name & ID. And these lines do just that.

Reply:
Nopes! Please change the message to 
```
error_message = stderr.write("ORGINIAL EXCEPTION: No such relation between Challenge Phase {} and DatasetSplit specified by Challenge Host \n {}".format(challenge_phase, split_code_name))
```